app/config/collections.php

```diff
@@ -1989,6 +2044,133 @@
+        'attributes' => [
+            [
+                '$id' => 'dateCreated',
+                'type' => Database::VAR_INTEGER,
+                'format' => '',
+                'size' => 0,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => 'runtime',
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 2048,
+                'signed' => true,
+                'required' => true,
+                'default' => '',
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => 'status',
+                'type' => Database::VAR_STRING, 
+                'format' => '',
+                'size' => 256,
+                'signed' => true,
+                'required' => true,
+                'default' => 'processing',
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => 'outputPath',
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 2048,
+                'signed' => true,
+                'required' => false,
+                'default' => '',
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => 'stderr',
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 16384,
+                'signed' => true,
+                'required' => false,
+                'default' => '',
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => 'stdout',
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 16384,
+                'signed' => true,
+                'required' => false,
+                'default' => '',
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => 'buildTime',
+                'type' => Database::VAR_INTEGER,
+                'format' => '',
+                'size' => 0,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => 'envVars',
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 16384,
+                'signed' => true,
+                'required' => false,
+                'default' => new stdClass(),
+                'array' => false,
+                'filters' => ['json'],
+            ],
+            [
+                '$id' => 'sourceType',
```

Comment:
Not sure what sourceType is for?

Reply:
cc @christyjacob4

Reply:
sourceType is supposed to remember what storage device was used when storing the file where as source remembers the path. You can see what it will contain here:
https://github.com/appwrite/appwrite/blob/5d9a9f53d82d464b4e1730d9883af97b562764d9/app/workers/builds.php#L93-L94